lib/services/firebase_service.dart

```diff
@@ -1097,7 +1139,7 @@ class FirebaseService extends IFirebaseService {
         .where('formName', isEqualTo: formName)
         .where('patientState', isEqualTo: patientState)
         .get()
-        .then((value) => value.docs.first.id)
+        .then((value) => value.docs.last.id)
```

Comment:
If want latest document created  should use orderBy()

Reply:
![Screen Shot 2564-05-11 at 22 30 27](https://user-images.githubusercontent.com/57941667/117842817-7dd08700-b2a8-11eb-9060-3780160296df.png)
@Thaweesak-sai  How I use .orderby() in this function
I try to used .orderBy('creation') and then return null value

Reply:
limit() might help too